lib/src/bloc/movies_bloc.dart

```diff
@@ -7,15 +7,28 @@ import '../resources/movie_repository.dart';
 class MoviesBloc {
   final _movieRepository = MovieRepository();
   final _moviesFetcher = StreamController<Movie>.broadcast();
+  final _moviesSearchFetcher = StreamController<Movie>.broadcast();
```

Comment:
This is unnecessary because you only use the _moviesFetcher Stream (When you call to trending movies and for search)

Reply:
It's ok, it's no necessary, but if we think in "another vendor that provides the search feature"....it's fine :)